core/src/main/java/org/apache/spark/shuffle/sort/UnsafeShuffleWriter.java

```diff
@@ -140,6 +143,9 @@ public UnsafeShuffleWriter(
     this.transferToEnabled = sparkConf.getBoolean("spark.file.transferTo", true);
     this.initialSortBufferSize = sparkConf.getInt("spark.shuffle.sort.initialBufferSize",
                                                   DEFAULT_INITIAL_SORT_BUFFER_SIZE);
+    this.inputBufferSizeInBytes = (int) sparkConf.getSizeAsKb("spark.shuffle.file.buffer", "32k") * 1024;
+    this.outputBufferSizeInBytes = (int) sparkConf.getSizeAsKb("spark.shuffle.unsafe.file.output.buffer",
```

Comment:
as you are modifying it, can you also move these 2 configs into `org.apache.spark.internal.config`?